components/generate/layer/CustomizeLayer.tsx

```diff
@@ -117,21 +111,21 @@ function CustomizeLayer() {
                             direction="row"
                             sx={{
                               width: 600,
-                              overflowX: 'scroll',
-                              overflowY: 'hidden',
-                              position: 'relative'
+                              overflowX: "scroll",
+                              overflowY: "hidden",
+                              position: "relative"
                             }}
                           >
                             {layer.assets.map((asset, j) => (
                               <Box
                                 sx={{
                                   mr: 1,
-                                  backgroundColor: 'white',
+                                  backgroundColor: "white",
                                   border: 1,
-                                  borderColor: '#e0e0e0',
+                                  borderColor: "#e0e0e0",
                                   width: 54,
-                                  display: 'grid',
-                                  placeItems: 'center'
+                                  display: "grid",
+                                  placeItems: "center"
```

Comment:
I think we can create a new constant file for all colors as you suggested next PR